README.md

```diff
@@ -63,7 +63,9 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link]()
+- [Live Demo Link](https://tvmaze-shows.onrender.com/)
+- [Video presentation Link](https://www.loom.com/share/fb3735edaccf43588daece96b8fb021e?sid=422bfc02-37e3-45a4-b7b1-6477f5917291)
+  
 
```

Comment:
- [ ] Kindly note that it is said in that in design guilines that you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

  - Typography: font face, size, and weight
  - Layout: composition and space between elements.
  
  you design should have one of the main color of te mplate app design
  
Your designe | -
------ | -
![image](https://github.com/SanaTameem/react-capstone-tvmaze/assets/65068771/9530402c-2caf-400a-8841-8e3625da94c1) ![image](https://github.com/SanaTameem/react-capstone-tvmaze/assets/65068771/79948b08-360c-4897-a9cf-37789a299235) |-
**The template** |-
![image](https://github.com/SanaTameem/react-capstone-tvmaze/assets/65068771/ffc4ab25-1631-4c5b-98c8-30e30c41ce97)
**The template** |-
![image](https://github.com/SanaTameem/react-capstone-tvmaze/assets/65068771/efd5c2b4-bd27-4562-9c4d-c884ed42dddd) | -